scripts/setup-k8s-master.sh.in

```diff
@@ -92,29 +107,18 @@ kubectl apply -f {{StorageClassUrl}}
 # Set up the network policy blocking the AWS metadata endpoint from the default namespace.
 kubectl apply -f {{NetworkPolicyUrl}}
 
-INSTANCE_ID=$(ec2metadata --instance-id)
-# Add this machine (master) to the load balancer for external access
-aws elb register-instances-with-load-balancer \
-  --load-balancer-name {{LoadBalancerName}} \
-  --instances ${INSTANCE_ID} \
-  --region {{Region}}
-
 # Use kubeadm's kubeconfig command to grab a client-cert-authenticated
 # kubeconfig file for administrative access to the cluster.
 KUBECONFIG_OUTPUT=/home/ubuntu/kubeconfig
 
-# TODO(@chuckha): --apiserver-advertise-address is resolved to an IP address.
-# We don't want this to happen because we need to use the ELB load balancer as the
-# api server address.
-# Instead we set the server by hand.
 kubeadm alpha phase kubeconfig user \
   --client-name admin \
-  --apiserver-advertise-address "${LB_DNS}" \
+  --apiserver-advertise-address "${LB_IPV4}" \
   >$KUBECONFIG_OUTPUT
 
 # This line sets the generated kubeconfig file's api server address to our loadbalancer.
 # This should be removed if kubeadm supports non-ip advertise addresses.
-KUBECONFIG="${KUBECONFIG_OUTPUT}" kubectl config set-cluster kubernetes --server="https://${LB_DNS}"
+# KUBECONFIG="${KUBECONFIG_OUTPUT}" kubectl config set-cluster kubernetes --server="https://${LB_DNS}"
```

Comment:
prefer delete over comment out unless there is some reason to save this inline and not in git history?

Reply:
Forgot to remove it, thanks!